src/lib.rs

```diff
@@ -48,11 +57,11 @@ impl NearGas {
     ///
     /// # Examples
     /// ```
-    /// use near_gas::*;
+    /// use near_gas::NearGas;
     ///
     /// let tera_gas = NearGas::from_tgas(5);
     ///
-    /// assert_eq!(tera_gas.as_gas(), 5 * ONE_TERA_GAS);
+    /// assert_eq!(tera_gas.as_gas(), 5 * NearGas::from_tgas(1).as_gas());
```

Comment:
Let's replace it with inline const:

```suggestion
    /// assert_eq!(tera_gas.as_gas(), 5_000_000_000_000);
```

P.S. Apply everywhere else